### PR TITLE
Add channels to HCO's package

### PR DESCRIPTION
**Pull request by rthallisey**
Channels will represent all the content for a release.  Upgrading will be changing the channel on your subscription to get the new content.

- Add Channels
- Ask for `QUAY_USERNAME` and `QUAY_PASSWORD` if they don't exist

### CODE REVIEW

**Comment by rthallisey** [comment]
Marking this WIP because I'm having a hard time getting the channels to show up in marketplace...

**Comment by rthallisey** [comment]
@lveyde try testing upgrade with this change

File: templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in [hunk 1]
```diff
@@ -1,4 +1,7 @@
 packageName: kubevirt-hyperconverged
 channels:
```
Comment (djzager) [inline templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in]: This seems so strange to me...the idea that you can't have alpha and stable channels. Mind blowing.

Comment (rthallisey) [reply templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in]: The issue is the quay app registry.  It doesn't support anything except incremental numeric releases e.g. 2.0.0, 2.0.1, 3.0.0.  This method would only work if you had a registry per release.

File: tools/quay-registry.sh [hunk 2]
```diff
@@ -9,18 +9,22 @@ CLUSTER="${CLUSTER:-OPENSHIFT}"
 MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
 PACKAGE="${PACKAGE:-hco-operatorhub}"
 APP_REGISTRY_NAMESPACE="${APP_REGISTRY_NAMESPACE:-kubevirt-hyperconverged}"
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+
+# Latest version from: https://quay.io/application/kubevirt-hyperconverged/hco-operatorhub
```
Comment (rwsu) [inline tools/quay-registry.sh]: I don't see anything at this URL.

**Comment by rwsu** [comment]
Just one comment. Otherwise lgtm.

File: tools/quay-registry.sh (hunk 2 above)
Comment (rthallisey) [reply tools/quay-registry.sh]: It's the old url.  Should be https://quay.io/application/kubevirt-hyperconverged/kubevirt-hyperconverged

**Comment by rthallisey** [comment]
@Coveralls there's no line of Go in here so you don't have to worry :).

**Pull request merged**